ci/scripts/clear_notebook_output_cells.sh

```
@@ -0,0 +1,26 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -gt 0 ]; then
+    NOTEBOOK_FILES="$@"
+else
+    NOTEBOOK_FILES=$(git ls-files "*.ipynb")
+fi
+
+for NOTEBOOK_FILE in ${NOTEBOOK_FILES[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)